File: proxy/tests/script_engine/body_clear.js
/// <reference path="../../script_libs/js/roxy.d.ts" />

/** @type {Extension} */
const bodyClear = {
  request(flow) {
    flow.request.body.clear();
  },
  response(flow) {
    flow.response.body.clear();
  }
}
globalThis.extensions = [bodyClear];

File: proxy/tests/script_engine/body_clear.lua
pcall(require, "../../script_libs/lua/roxy.lua")
---@type Extension
local ext = {
	request = function(flow)
		flow.request.body.clear()
	end,

	response = function(flow)
		flow.response.body.clear()
	end,
}
Extensions = { ext }
